File: malevich-web/src/app/_transfer/paymentsDto.ts
import {PaymentMethodDto} from './paymentMethodDto';
import {PaymentTypeDto} from './paymentTypeDto';
import {ParticipantDto} from "./participantDto";

export class PaymentsDto {
  id: number;
  participant: ParticipantDto;
  effectiveDate: Date;
  paymentMethod: PaymentMethodDto;
  amount: number;
  paymentType: PaymentTypeDto;
  transactionId: string;
}

File: malevich-server/src/main/java/io/malevich/server/transfer/PaymentsDto.java
package io.malevich.server.transfer;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class PaymentsDto {


    private Long id;

    private ParticipantDto participant;

//    @JsonFormat(pattern = "dd/MM/yyyy")
    private java.sql.Timestamp effectiveDate;

    private PaymentMethodSuperDto paymentMethod;

    private Double amount;

    private PaymentTypeDto paymentType;

    private String transactionId;

}
